packages/gateway/src/types.ts

```diff
@@ -39,3 +39,32 @@ export type GetTextProps = {
 export interface Signer {
   sign: (msg: SignableMessage) => Promise<Hex>
 }
+
```

Comment:
As the added code is related to an interface, not a type, maybe it will be better if we create another file called interfaces.ts to put it.